rootfs/usr/bin/steamos-session-select

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/bash
+
+set -e
+
+die() { echo >&2 "!! $*"; exit 1; }
+
+CONF_FILE="/etc/lightdm/lightdm.conf.d/10-chimeraos-session.conf"
+SENTINEL_FILE="steamos-session-select"
+
+# For sanity this shipped file must be present, to ensure we're still on a normal-looking Chieraos setup.
+CHECK_FILE="/etc/lightdm/lightdm.conf.d/10-chimeraos-session.conf"
+
+session="${1:-gamescope}"
+
+session_launcher=""
+create_sentinel=""
+
+
+# We use "plasma" as "desktop" to hook up to SteamOS's scripts
+case "$session" in
+  plasma)
+    session_launcher="gnome-wayland"
```

Comment:
When we run `chimera-session desktop` I believe we currently use gnome X11. Should that be unified?
I also recall having trouble getting the gnome wayland to work properly.

Reply:
Wayland depends mostly on driver support... Nvidia could have a bad time(tm) but it mostly works for me on my laptop. The issuel with X11 is it doenst work well with touchscreens. I figured if gamescope session works, gnome-wayland should definitely work. Remember this switcher is only meant to be for gamepadui and back but it can be adapted to work based on a sessing on which session to use based on a config file (didn't do it because I'm lazy :P ).

Reply:
Ah, good point.